pkg/registry/apis/folders/register.go

```diff
@@ -189,7 +189,7 @@ func (b *FolderAPIBuilder) GetAuthorizer() authorizer.Authorizer {
 func authorizerFunc(ctx context.Context, attr authorizer.Attributes) (*authorizerParams, error) {
 	verb := attr.GetVerb()
 	name := attr.GetName()
-	if (!attr.IsResourceRequest()) || (name == "" && verb != utils.VerbCreate) {
+	if (!attr.IsResourceRequest()) || (name == "" && verb != utils.VerbCreate && verb != utils.VerbList) {
```

Comment:
We have some tests for the authoriser that we can extend here to include list use case: https://github.com/grafana/grafana/blob/main/pkg/registry/apis/folders/register_test.go#L25

Reply:
That's true! I added some tests for List Folders there!